### PR TITLE
Add dropdown to enable drawing shapes with fixed aspect ratios/sizes

### LINKED ISSUE #26: Need ability to create fixed-size rectangular annotations
This is useful for creating things like regions of interest (ROI) where size may need to be fixed instead of being determined by mouse dragging.

### PR DESCRIPTION
**Pull request by mvandenburgh**
Closes #26

### CODE REVIEW

File: histomicsui/web_client/panels/DrawWidget.js [hunk 1]
```diff
@@ -756,6 +766,38 @@ var DrawWidget = Panel.extend({
...
+            fixed_height: parseFloat(this.$('.h-fixed-height').val())
+        };
+        this._saveEditOptions(opts);
+
+        this._updateConstraintValueInputs();
+
+        if (opts.size_mode === 'fixed_aspect_ratio') {
+            this.viewer.startDrawMode(this._drawingType, {constraint: opts.fixed_width / opts.fixed_height});
+        } else if (opts.size_mode === 'fixed_size') {
+            this.viewer.startDrawMode(this._drawingType, { constraint: { width: opts.fixed_width, height: opts.fixed_height } });
```
Comment (mvandenburgh) [inline histomicsui/web_client/panels/DrawWidget.js:797]: @manthey this isn't finished, I still have some minor styling issues to iron out and also have to make this work with ellipses as well (it currently only works with rectangles). But, I'm running into the same problem I mentioned [here](https://github.com/DigitalSlideArchive/HistomicsUI/issues/26#issuecomment-1144087622).
I tried specifying a constraint like you said [here](https://github.com/DigitalSlideArchive/HistomicsUI/issues/26#issuecomment-1126089848), but it doesn't seem to work. The drawing tool behaves the same as it normally does for me, no matter what the `constraint` option is set to. Any ideas?

**Comment by manthey** [comment]
We weren't passing the mode options through.  I've made a PR in large_image to do this: https://github.com/girder/large_image/pull/881
You'll need to change the code to something like:
```
const options = {modeOptions: {}};
options.modeOptions.constraint = { width: this._editOptions.fixed_width, height: this._editOptions.fixed_height };
```

**Comment by manthey** [comment]
I noticed that the dropdown arrows on the control panels were inconsistent because of how I had implemented them.  I pushed a change to fix that.

Otherwise, this is working for me.  I'll approve this -- if you don't like my changes, feel free to un-approve and handle things a different way.

**Review by manthey — APPROVED** [review]

**Comment by manthey** [comment]
I think you'll be able to merge without the coverage test passing.

**Pull request merged**